CMakeLists.txt

```diff
@@ -94,12 +94,12 @@ if( APPLE )
    set( MACOS_ARCHITECTURE "${HOST_SYSTEM_PROCESSOR}" CACHE STRING "Target architecture for macOS builds. One of x86_64/arm64" )
 
    if( MACOS_ARCHITECTURE STREQUAL "x86_64" )
-      set( MIN_MACOS_VERSION 10.9 )
```

Comment:
@crsib I don't know the consequences of this, but that'd be the easiest fix for me :D So tentatively proposing it.

Reply:
Does that mean users is MacOS < 10.15 wouldn't be able to run newer Audacity releases ?

Reply:
This will break macOS. We cannot have a target greater than 10.13.

Also, raising a target requires a lot of thought and discussion. We have plenty users on older macOS versions

Reply:
That's all right, I gave up on using (experimental/)filesystem to fix this.